linera-service/src/storage.rs

```diff
@@ -318,6 +318,30 @@ impl FullStorageConfig {
             }
         }
     }
+
+    /// List all the tables of the database
+    pub async fn list_tables(self) -> Result<Vec<String>, ViewError> {
+        match self {
+            FullStorageConfig::Memory(_) => Err(ViewError::ContextError {
+                backend: "memory".to_string(),
+                error: "delete_single does not make sense for memory storage".to_string(),
```

Comment:
"list_tables is not supported for the memory storage"

Reply:
Done.